flagconf.go

```diff
@@ -122,6 +123,8 @@ and the types supported by the flag package. The basic types are therefore:
     uint64
     float64
 
+For convenience, flagconf also supports any type implementing flag.Value, as long as TOML also supports it.
```

Comment:
This section now reads a little strangely ("flagconf only supports these...but also this...but also this other exception"). I'll reword after I merge though.